iOS/Settings/Appearance/ArticleThemeManagerView.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  ArticleThemeManagerView.swift
+//  NetNewsWire-iOS
+//
+//  Created by Stuart Breckenridge on 20/12/2022.
+//  Copyright © 2022 Ranchero Software. All rights reserved.
+//
+
+import SwiftUI
+
+struct ArticleThemeManagerView: View {
+    
+	@StateObject private var themeManager = ArticleThemesManager.shared
+	@State private var showDeleteConfirmation: (String, Bool) = ("", false)
+	@State private var showImportThemeView: Bool = false
+	@State private var showImportConfirmationAlert: (ArticleTheme?, Bool) = (nil, false)
+	@State private var showImportErrorAlert: (Error?, Bool) = (nil, false)
+	@State private var showImportSuccessAlert: Bool = false
+	
+	var body: some View {
```

Comment:
I was playing around with a preview for the theme manager
Let me know if you want to connect over it and put it in:
![Simulator Screen Shot - iPhone 14 - 2023-02-22 at 15 46 33](https://user-images.githubusercontent.com/1043281/220755245-3e7a848b-d81e-4e86-a2e0-cc3f808d095a.png)